forms-flow-api/src/formsflow_api/services/influenza_service.py

```diff
@@ -0,0 +1,60 @@
+import requests
+from flask import current_app
+
+class InfluenzaService:
+    """This class manages Influenza form related service."""
+
+    @staticmethod
+    def get_ministry_names():
+        """fetch ministry_names"""
+        try:
+            ministry_api_url = "{}/ODS_Datamart_influenza_ministry".format(current_app.config.get("ODS_URL"))
+            response_from_ods = requests.get(ministry_api_url,
+                        headers={"Authorization": current_app.config.get("ODS_AUTH_TOKEN")})
+        except Exception as e:
+            current_app.logger.error(e)
+            return []
```

Comment:
Returns `[]` in case of an exception might confuse the client. Instead, let's raise an error and catch it in the resource class, so we can send a more helpful message to the client. Check out the EmployeeData resource and service files for examples of how to do this.

Reply:
@iman-jamali-fw 
`[]` set intentionally. as in response If other than 200 code it stopped working in form Select Component.

Reply:
Can you move that logic to the client side, in this case, the Select component to handle the error?